tasks/main.yml

```diff
@@ -48,6 +48,30 @@
     (ansible_os_family != "Darwin" and nix_directory is changed) or
     (ansible_os_family == "Darwin" and not nix_directory_mac.stat.exists)
 
+- name: Create /nix bind mount
+  when:
+    - ansible_os_family != "Darwin"
+    - nix_bind_mount != ""
+  block:
+    - name: Create bind mount target directory
+      become: true
+      ansible.builtin.file:
+        path: "{{ nix_bind_mount }}"
+        owner: "{{ nix_user }}"
+        state: directory
+        mode: "0755"
+      register: nix_bind_mount_directory
+
+    - name: "Bind mount /nix to {{ nix_bind_mount_directory.path }}"
+      become: true
+      ansible.posix.mount:
+        path: "{{ nix_bind_mount_directory.path }}"
+        src: "/nix"
+        opts: bind
+        state: mounted
+        fstype: none
+      when: nix_bind_mount_directory is changed  # noqa: no-handler
```

Comment:
What if `/nix` already existed, thus `nix_bind_mount_directory` is not `changed`?